pyomo/contrib/pynumero/sparse/tests/test_mpi_block_vector.py

```diff
@@ -169,6 +169,28 @@ def test_min(self):
         self.assertEqual(self.v1.min(), 0.0)
         self.assertEqual(self.v2.min(), 0.0)
 
+    def test_min_max_with_size0_blocks(self):
```

Comment:
Can we also add a test to exercise the exceptions for empty vectors?

Reply:
Seconded but this looks good otherwise.